docs/source/installation.rst

```diff
@@ -102,6 +102,21 @@ like that yourself:
 
    installation/jupyter
 
+.. _editor-addons:
+
+Editors
+********
+
+If you're using Visual Studio Code you can install an extension called
+`Manim Sideview` which provides automated rendering and an integrated preview
+of the animation inside the editor.
+
+The extension can be installed through the marketplace or by pressing install `here
+<https://marketplace.visualstudio.com/items?itemName=Rickaym.manim-sideview>`__.
+
+Further instruction and support regarding the extension can be found in its
+`repository <https://github.com/Rickaym/Manim-Sideview>`__ or the marketplace webpage.
```

Comment:
Three paragraphs seems a bit much, allow me to make a suggestion for shortening this (while still keeping all relevant information):

```suggestion
of the animation inside the editor. The extension can be installed through the
`marketplace of VS Code <https://marketplace.visualstudio.com/items?itemName=Rickaym.manim-sideview>`__.
```